run-periodically.js

```diff
@@ -3,11 +3,11 @@ const createCoursesFile = require('./server/createCoursesFile')
 const createEnrollmentsFile = require('./server/createEnrollmentsFile')
 const CanvasApi = require('kth-canvas-api')
 const schedule = require('node-schedule')
-const canvasApi = new CanvasApi(process.env.canvasApiUrl, process.env.canvasApiKey)
+const canvasApi = new CanvasApi(process.env.CANVAS_API_URL, process.env.CANVAS_API_KEY)
 canvasApi.logger = logger
 const moment = require('moment')
 const createSectionsFile = require('./server/createSectionsFile')
-const cronTime = process.env.successfulSchedule || '2 * * *'
+const cronTime = process.env.SUCCESFUL_SCHEDULE || '2 * * *'
```

Comment:
To avoid potentially running every minute of the hour 2, this default value should be set to `0 5 * * *` imo.

Reply:
Will update to aforementioned value.